### PR TITLE
Remove dependency on test/integration from kubemark

### PR DESCRIPTION
**Pull request by gmarek**
Fixes kubemark builds broken by #27034

### CODE REVIEW

File: test/utils/tmpdir.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*
```
Comment (wojtek-t) [inline test/utils/tmpdir.go]: The formatting is strange in this header.

Comment (gmarek) [inline test/utils/tmpdir.go]: Done. I think.

**Comment by wojtek-t** [comment]
LGTM

@k8s-oncall - we should merge on green to fix tests.

**Comment by gmarek** [comment]
Merging to fix the build.

**Comment by fabioy** [comment]
Thanks for fixing this. Waiting for latest kubemark run (and hoping it goes green).

**Pull request merged**